experimental-features.md

```diff
@@ -0,0 +1,62 @@
+---
+title: TiDB Experimental Features
+summary: Learn the experimental features of TiDB.
+aliases: ['/tidb/dev/experimental-features-4.0/']
+---
+
+# TiDB Experimental Features
+
+This document introduces the experimental features of TiDB in different versions. It is **NOT** recommended to use these features in the production environment.
+
+## Stability
+
++ TiFlash limits the use of I/O resources by compressing or sorting data, mitigating the contention for I/O resources between background tasks and front-end data reading and writing (Introduced in v5.0)
++ Improve the stability of the optimizer's choice of indexes (Introduced in v5.0)
+    + Extend the statistics feature by collecting the multi-column order dependency information.
+    + Refactor the statistics module, including deleting the `TopN` value from `CMSKetch` and the histogram, and adding NDV information for histogram buckets of each table index.
++ [hibernate-regions](/tikv-configuration-file.md#hibernate-regions-experimental). A Region that idles for a long time is automatically set as hibernated. This reduces the extra overhead caused by heartbeat messages between the Raft leader and the followers for idle Regions.
+
```

Comment:
This seems to missing:
* https://docs.pingcap.com/tidb/v5.0/system-variables#tidb_enable_async_commit-new-in-v500-rc

Reply:
This document has been updated. Please see the latest version in the preview environment I gave you earlier.

Reply:
I meant it is missing from the experimental list here.

Reply:
I meant in the latest version, `tidb_enable_async_commit` is not experimental.

Reply:
Ahh okay! Thanks.

Reply:
No problem.